index.html

```diff
@@ -4,24 +4,20 @@
    <meta charset="UTF-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-   <title>Toshia_moi_mentor_dorogoi</title>
+   <title>website</title>
    <link href="src/bootstrap/bootstrap.min.css" rel="stylesheet">
    <link rel="preconnect" href="https://fonts.gstatic.com">
    <link href="https://fonts.googleapis.com/css2?family=Open+Sans:ital,wght@0,400;0,700;1,400&display=swap" rel="stylesheet"> 
    <link rel="stylesheet" type="text/css" href="dist/main.css">
 </head>
 <body>
-   <header class="header">
-      <div class="header-top">
-         <nav class="nav">
-            <ul class="nav-list">
-               <li class="nav-list-item"><a class="nav-list-link" href="#">HOME</a></li>
-               <li class="nav-list-item"><a class="nav-list-link" href="#">PORFOLIO</a></li>
-               <li class="nav-list-item"><a class="nav-list-link" href="#">ABOUT US</a></li>
-               <li class="nav-list-item"><a class="nav-list-link" href="#">CONTACT</a></li>
-            </ul>
-        </nav>
-      </div>
-   </header>
+   <nav class="nav">
```

Comment:
you can make it even smaller, `nav.nav -> a.nav-item`, that's enough

Reply:
done